Change QPad to qStudio (#101)
DataIntellectTech/TorQ-Amazon-FinSpace-Starter-Pack: docs/10-connectingusingqStudio.md

Connecting Using qStudio
===============

> Open qStudio, right-click on Servers in the top left corner and select 'Add Server'.

Reviewer: ```suggestion
Open qStudio, right-click on 'Servers' in the top left corner and select 'Add Server'.
```

![Add Server Connection](workshop/graphics/Servers_qStudio1.png)

> You will then see a pop-up showing the server properties. This is where you will add the information from the connection string.

Reviewer: ```suggestion
You will then see a pop-up showing the server properties. This is where you will add the information from the connection string (details below in the next step).
```

![Server Properties](workshop/graphics/Server_Properties1.png)

> The generated connection string provides you with everything you need to fill this out, take this connection string for example;

Reviewer: ```suggestion
Follow the step in our [Generate Connection String section](https://dataintellecttech.github.io/TorQ-Amazon-FinSpace-Starter-Pack/08-clusterconnectionstring/#generate-connection-string) to get a fresh connection string. The generated connection string provides you with everything you need to fill the boxes above. See the below connection string for an example:
```
